js/freewheel.js

```diff
@@ -464,28 +461,6 @@ OO.Ads.manager((function(_, $) {
     }, this);
 
     /**
-     * Called by the ad manager controller.  Pauses the current ad.
-     * @public
-     * @method Freewheel#pauseAd
-     * @param {object} ad The ad to pause
-     */
-    this.pauseAd = function(ad) {
-      OO.log("FW: Pausing the ad");
-      amc.ui.adVideoElement[0].pause();
-    };
-
-    /**
-     * Called by the ad manager controller.  Resumes the current ad.
-     * @public
-     * @method Freewheel#resumeAd
-     * @param {object} ad The ad to resume
-     */
-    this.resumeAd = function(ad) {
-      OO.log("FW: Resuming the ad");
-      amc.ui.adVideoElement[0].play();
-    };
-
-    /**
```

Comment:
I noticed that the pause and resume functions are removed from Freewheel, but remain in Vast with an empty implementation and a comment saying VTC will handle them. Should we keep the functions in Freewheel and have a comment saying why there is no implementation? Just to get more information visible. Minor thing.